src/components/navbar/navbar.tsx

```diff
@@ -0,0 +1,21 @@
+import Logo from '../logo/logo';
+import Button from '../reusable/buttons';
+function navbar() {
```

Comment:
use arrow functions and start the component name with capital letter. Also name this file as `index.tsx` since this is the entry point component. Do the same for all created components

Reply:
done